elements/pfe-textinput/src/pfe-textinput.json

```
@@ -0,0 +1,44 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Textinput",
+  "description": "A web component text input",
+  "type": "object",
+  "tag": "pfe-textinput",
+  "class": "PfeTextinput",
+  "category": "content",
+  "properties": {
+    "slots": {
+      "title": "Slots",
+      "description": "Definition of the supported slots",
+      "type": "object",
+      "properties": {
+        "default": {
+          "title": "Default slot",
+          "type": "array",
+          "namedSlot": false,
+          "items": {
+            "oneOf": [
+              {
+                "$ref": "raw"
```

```suggestion
                "$ref": "input"
```

#pebble This doesn't actually change anything about the logic but since only inputs are allows, 🤷 .  `pfe-cta` is kind of doing this I think where it specifies what can go in the slot this way.